.github/workflows/ci.yml

```
@@ -78,17 +78,19 @@ jobs:
       fail-fast: false
       matrix:
         php: ['7.4', '8.0', '8.1', '8.2']
-        symfony_version: ['5.4.*', '6.2.*', '6.3.*']
+        symfony_version: ['5.4.*', '6.4.*', '7.0.*']
         dependencies: ['--prefer-lowest', '--prefer-dist']
         exclude:
           - php: '7.4'
-            symfony_version: '6.2.*'
+            symfony_version: '6.4.*'
           - php: '7.4'
-            symfony_version: '6.3.*'
+            symfony_version: '7.0.*'
           - php: '8.0'
-            symfony_version: '6.2.*'
+            symfony_version: '6.4.*'
           - php: '8.0'
-            symfony_version: '6.3.*'
+            symfony_version: '7.0.*'
+          - php: '8.1'
```

Symfony 7 requires PHP 8.2 or upper: https://symfony.com/releases/7.0

```suggestion
          - php: '8.2'
```

---

This is the exclusion matrix, which points to PHP version that should **not** be run with this Symfony version. I think this is the correct setting here if I'm not missing something